definitions/npm/stringify-object_v3.x.x/flow_v0.50.x-/stringify-object_v3.x.x.js

```diff
@@ -0,0 +1,17 @@
+declare module 'stringify-object' {
+  declare type AnyObject = { [key: mixed]: mixed };
```

Comment:
Any reason to use the indexer property here? I think `{}` would suffice.

Reply:
That's what I got first but the `obj[prop]` gives error.

```
Cannot access computed property using mixed [1].

      test_stringify-object_3.x.x.js:64:37
      64|   filter: (obj, prop) => typeof obj[prop] === 'string',
                                              ^^^^

   References:
      stringify-object_v3.x.x.js:7:30
       7|     filter?: (obj: {}, prop: mixed) => boolean,
                                       ^^^^^ [1]
```

Happy to get insights on this.

Reply:
Done.